Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -118,6 +126,16 @@ class OverviewViewController: UIViewController {
         MultiWalletSyncDetailsLoader.setup(for: self.multipleWalletsSyncDetailsTableView)
     }
     
+    @objc func refresh() {
+        self.refreshControl.beginRefreshing()
+        defer {
+            self.recentTransactionsTableView.reloadData()
+            self.refreshControl.endRefreshing()
+        }
```

Comment:
No need for a defer, `reloadData()` is already called in `updateRecentActivity()`.
Code order should be
```
self.refreshControl.beginRefreshing()
self.updateMultiWalletBalance()
self.updateRecentActivity()
self.refreshControl.endRefreshing()
```

Reply:
resolved in `698f737 `